llvm/docs/CommandGuide/llvm-objdump.rst

```diff
@@ -291,6 +291,36 @@ OPTIONS
 
   Target triple to disassemble for, see ``--version`` for available targets.
 
+.. option:: --visualize-jumps=<modes>
+
+  Display a control-flow graph which shows the targets of branch instructions to the left of disasembly. ``modes`` is a comma-separated list of options, which configure the character set and used to print the graph.
+
+  If ``modes`` is omitted, the default is ``unicode,auto``
```

Comment:
Unlike `.td` help messages, the documentation descriptions should have a trailing period.